### PR TITLE
Fix NSURLProtocol_WinHTTP file to use the updated NSHTTPURLResponse initializer

### PR DESCRIPTION
**Pull request by mnithish**

### CODE REVIEW

File: Frameworks/Foundation/NSURLProtocol_WinHTTP.mm [hunk 1]
```diff
@@ -377,10 +378,23 @@ - (HRESULT)_asyncHttpOperation:(AsyncHttpOperation*)pOperation completedWithStat
...
 
-        NSHTTPURLResponse* nsResponse = [[[NSHTTPURLResponse alloc] initWithURL:[_request URL]
-                                                                     statusCode:statusCode
-                                                                        headers:nsHeaders
-                                                          expectedContentLength:contentLength] autorelease];
+        NSString* httpVersionStr;
+        switch (httpVersion) {
+            case HttpVersion_Http10:
+                httpVersionStr = [[[NSString alloc] initWithString:@"HTTP/1.0"] autorelease];
+                break;
```
Comment (DHowett-MSFT) [inline Frameworks/Foundation/NSURLProtocol_WinHTTP.mm]: prefer `httpVersionStr = @"HTTP/1.0"` (for all of these)

**Comment by mnithish** [comment]
@jo is added to the review. #Closed

**Comment by mnithish** [comment]
@jofre-ms is added to the review. #Closed

**Comment by DHowett-MSFT** [comment]
:clock1:

**Comment by rajsesh-msft** [comment]
:shipit:

**Comment by mnithish** [comment]
CI build successful

**Comment by jofre-ms** [comment]
:shipit:

**Pull request merged**